### DISCUSSION TITLE
Can't import Scroll Area component

### DISCUSSION THREAD

**Discussion by Miloszzz2** [huntabyte/shadcn-svelte#913] [Help]
I have an issue with Scroll Area component. This error appears in scroll-area.svelte file:
![image](https://github.com/huntabyte/shadcn-svelte/assets/97192271/79af4978-ccc7-467f-97b9-45f85dc79864)

I've tried to update/reinstall bits-ui package but it didn't help

**Comment by shyakadavis** [comment]
Hi;

What version of `bits-ui` are using?

**Reply by Miloszzz2** [reply]
Hi,
I've tried with that one installed by  default by shadcnui and then with the latest one and still getting same error

**Reply by shyakadavis** [reply]
Could you send the exact version number? Or a small repro?!

**Reply by Miloszzz2** [reply]
0.19.7
